frontend/src/components/common/SideBar/style.ts

```diff
@@ -17,10 +17,12 @@ interface PositionerProps {
   isExpanded: boolean;
 }
 
+export const SIDEBAR_BOX_SHADOW_COLOR = "rgba(16, 22, 26, 0.15)";
```

Comment:
this color will be used as a separator line for the split right sidebar, top and bottom views